src/coreclr/tools/Common/Internal/Runtime/CorConstants.cs

```diff
@@ -95,4 +95,36 @@ public enum CorTokenType
         mdtName = 0x71000000,
         mdtBaseType = 0x72000000,
     }
+
+    [Flags]
+    public enum CorUnmanagedCallingConvention
```

Comment:
These constants are defined by System.Reflection.Metadata as `SignatureCallingConvention` and `SignatureAttributes`. We can just use those definitions instead of having yet another copy.

Reply:
Thanks Jan for pointing that out, I have reverted the additions to CorConstants and modified the code to reuse the pre-existing constants.